debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/IncrementalSnapshotIT.java

```diff
@@ -56,7 +56,9 @@ public void after() {
                 .with(PostgresConnectorConfig.SIGNAL_DATA_COLLECTION, "s1.debezium_signal")
                 .with(PostgresConnectorConfig.INCREMENTAL_SNAPSHOT_CHUNK_SIZE, 10)
                 .with(PostgresConnectorConfig.DROP_SLOT_ON_STOP, false)
-                .with(RelationalDatabaseConnectorConfig.MSG_KEY_COLUMNS, "s1.a42:pk1,pk2,pk3,pk4");
+                .with(RelationalDatabaseConnectorConfig.MSG_KEY_COLUMNS, "s1.a42:pk1,pk2,pk3,pk4")
+                // DBZ-4272 required to allow dropping columns just before an incremental snapshot
+                .with("database.autosave", "conservative");
```

Comment:
Should we mention this in the connector documentation?

Reply:
@jpechane, @Naros, ok, so my understanding is, this option is _only_ needed to enable an incremental snapshot right after a schema change (i.e. no change event has been coming through in between). Without that setting, the incremental snapshot would fail in that case, and the user would have to restart it (once there has been a change event). Is that correct? If so, no need to document this very rare situation, IMO.